src/web/app/components/adding-question-panel/adding-question-panel.component.html

```diff
@@ -5,10 +5,15 @@
       <button class="dropdown-item" (click)="templateQuestionModalHandler()">Choose from template questions</button>
       <div class="dropdown-divider"></div>
       <ng-container *ngFor="let questionType of FeedbackQuestionType | enumToArray">
-      <button class="dropdown-item" *ngIf="questionType !== FeedbackQuestionType.CONSTSUM"
+        <div *ngIf="questionType !== FeedbackQuestionType.CONSTSUM">
+          <div class="d-flex btn-group justify-content-between" role="group">
+              <button class="dropdown-item" *ngIf="questionType !== FeedbackQuestionType.CONSTSUM"
```

Comment:
I believe the `*ngIf="questionType !== FeedbackQuestionType.CONSTSUM"` is not needed for the button element since it is already wrapped in a div that ensures that the question type is not constant sum

Reply:
You are right, will make the change.